### DISCUSSION TITLE
my computer is shit at exporting fnf files

### DISCUSSION THREAD

**Discussion by daveandbambifan1234** [JordanSantiagoYT/FNF-JS-Engine#799] [General]
![image](https://github.com/user-attachments/assets/c0bc0cbe-5b7e-4a27-947e-eed21c64202b)

**Comment by NAEL2XD** [comment]
off topic moment;

Anyways 2 things:
- it happens if something has the same name including the file extension so it'll slow down a bit for it
- yeah I think your computer is bad
